projects.go

```diff
@@ -236,6 +246,31 @@ func (s *ProjectsService) ListUserProjects(uid interface{}, opt *ListProjectsOpt
 	return p, resp, err
 }
 
+// ListProjectsUsers gets a list of users for the given project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/projects.html#get-project-users
+func (s *ProjectsService) ListProjectsUsers(uid interface{}, opt *ListProjectsOptions, options ...OptionFunc) ([]*ProjectUsers, *Response, error) {
```

Comment:
This function should not take `opt *ListProjectsOptions`. So this line should just use a `nil`, instead of `opt`:

```go
req, err := s.client.NewRequest("GET", u, nil, options)
```

Reply:
With ```opt *ListProjectsOptions``` we have more control for what we may need, if a project have 200 users, we can use pagination, sorting, ordering and so on in an easy way, but at least for what I'm using, ```nil``` aren't a problem, so any one of then are good for me.

Reply:
Ah, check... But in that case you should make a option struct dedicated for this call. That would be fine too.

Reply:
Done, thanks for the feedback!